webapp/apps/taxbrain/views.py

```diff
@@ -1014,7 +1014,7 @@ def csv_output(request, pk):
     filename = "taxbrain_outputs_" + suffix + ".csv"
     response['Content-Disposition'] = 'attachment; filename="' + filename + '"'
```

Comment:
It is probably better to use the following:

```
filename = 'taxbrain_outputs_{}.csv'.format(suffix)
response['Content-Disposition'] = 'attachment; filename="{}"'.format(filename)
```

Also: For debugging purposes, it is a good idea to [log things](https://docs.djangoproject.com/en/2.0/topics/logging/) (the response filename, for example).